app/controllers/categories_controller.rb

```diff
@@ -1,9 +1,11 @@
 class CategoriesController < ApplicationController
 	def index
 		categories_all = Category.all
+		# users = User.where(id: current_user.id)
 		lessons = Lesson.where(user_id: current_user.id)
```

Comment:
@YuheiTsunoda  Remove unnecessary code.
Change this line to : `lessons = current_user.lessons`